front/app/components/admin/ImageCropper/Container.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Text, Box } from '@citizenlab/cl2-component-library';
+import Warning from 'components/UI/Warning';
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import ImageCropper, { ImageCropperProps } from '.';
+import messages from './messages';
+
+const Container = ({ aspect, ...otherProps }: ImageCropperProps) => {
+  const { formatMessage } = useIntl();
+  return (
+    <Box>
```

Comment:
Text was here before, I didn't want to break anything I could miss. And I suppose it doesn't hurt.

I've used Box instead of `<>` to prevent this bug in the future https://github.com/CitizenLabDotCo/citizenlab/pull/3604
Maybe not a very strong reason, but again it doesn't hurt.